client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,138 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptySites = styled( EmptyContent )`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+`;
+
+const SecondaryText = styled.p`
+	max-width: 650px;
+`;
+
+const Title = styled.div`
+	margin-top: 50%;
+`;
+type SitesContainerProps = {
+	sites: SiteData[];
+	status: string;
+};
+
+export const SitesContainer = ( { sites, status }: SitesContainerProps ) => {
+	const { __ } = useI18n();
+	sites = [];
```

Comment:
Looks like this `sites = [];` needs to be removed 😄

Reply:
😆 that's why we have code review 🤣